src/components/small-item-table/index.js

```diff
@@ -365,6 +365,8 @@ function SmallItemTable(props) {
                         if (prev.priceRUB > current.priceRUB) return prev;
                         return current;
                     }, {priceRUB: 0})
+                } else {
+                    formattedItem.bestSell = formattedItem.bestSell[0];
```

Comment:
If length is 0 we will access a null pointer

Reply:
Good point; fixed!